src/components/commons/Header.tsx

```diff
@@ -7,6 +7,9 @@ import HomeHeaderContent from './HomeHeaderContent';
 import NavHeaderContent from './NavHeaderContent';
 import { LogoIcon } from '../../assets/svgs';
 
+type scroll = {
+  isScroll: boolean;
+};
```

Comment:
우리 prop 네이밍 컨벤션이랑 받아올 때 컨벤션 있슴니다 ! 
```suggestion
interface HomeDefaultHeaderProps = {
  isScroll: boolean;
};
```

![스크린샷 2024-05-22 오전 3 19 01](https://github.com/NOW-SOPT-CDSP-TEAM-WEB2/Client/assets/109661444/d798dcc3-3aec-4aeb-bdcd-af499fa9e5e3)

Reply:
앗 요 컨벤션이 있는걸 늦게 깨달아서 고치고 올려야지! 했는데 까먹고 안고치고 올렸네요... 핳 다고쳐놓겠습니다!!